website/src/css/custom.css

```diff
@@ -1,4 +1,19 @@
-/* your custom css */
+/**
+ * /src/css/custom.css
+ * You can override the default Infima variables here.
+ * Note: this is not a complete list of --ifm- variables.
+ */
+ :root {
+    --ifm-color-primary: #75c7a4;
+    --ifm-color-primary-dark: rgb(105, 173, 147);
+    --ifm-color-primary-darker: rgb(99, 163, 139);
+    --ifm-color-primary-darkest: rgb(82, 134, 115);
+    --ifm-color-primary-light: rgb(138, 201, 178);
+    --ifm-color-primary-lighter: rgb(158, 211, 191);
+    --ifm-color-primary-lightest: rgb(186, 224, 210);
+  }
+
+  /* your custom css */
 
 @font-face {
   font-family: 'Lato';
```

Comment:
I don't think the Lato Google font is loaded at all...

Reply:
Yes I'd lean towards that. We won't need to write too much in `custom.css` as they apply globally. Page-specific CSS should be written in CSS modules.

Reply:
@yangshun please check for the updated build here  https://thakurkarthik.github.io/redex/ .In my latest commit i have moved styles to local module.In the custom.css only some colors and fonts are present,for the fonts i am not that much aware if they are working or not.Can you please check for that and signal me accordingly.Thanks for the reviews !

Reply:
The fonts look different from those on the current fbredex.com...